modules/javafx.graphics/src/main/java/javafx/stage/PopupWindow.java

```diff
@@ -458,12 +458,10 @@ private void showImpl(final Window owner) {
         SceneHelper.parentEffectiveOrientationInvalidated(sceneValue);
 
         // RT-28447
+        applyStylesheetFromOwner(owner);
```

Comment:
minor: Does this comment RT-28477 (https://bugs.openjdk.org/browse/JDK-8116444) applies to the change of to the code that follows?

Reply:
Either here or in the new method I think. What is your opinion?

Reply:
I wasn't sure; I suppose you have it right.

I would have added the new JBS ticket id since RT- prefix is no more, but it is ok as it is.

Reply:
Changed to the JDK number, in case this lookup is needed in the future

Reply:
thank you